### ISSUE TITLE
The new save new extension option doesn't seem to work anymore

### ISSUE THREAD

**Issue by Vahvl** [Nordgaren/Erd-Tools-CPP#6]
title says it all, it just uses the default save extension.

**Comment by Vahvl** [comment]
nevermind i did smth wrong lmao

**Comment by Vahvl** [comment]
okay it is still broken, but occasionally works somewhy

**Comment by Nordgaren** [comment]
Did you try using the load order feature of elden mod loader with load order 0?

**Comment by Vahvl** [comment]
yeah setting it lower than 5000 fixed it, i just left it at 2500, thanks
